### ISSUE TITLE
Add support for Etcd v3

### ISSUE THREAD

**Issue by falenn** [hpe-storage/python-hpedockerplugin#733]
Please add support for use with etcd v3.

**Comment by falenn** [comment]
Looking forward at k8s 1.15+, the ecosystem has migrated to coreos/etcd 3.4+ presenting the new v3 API.  As far as I can tell, hpedockerplugin and doryd should follow.

**Comment by wdurairaj** [comment]
Yes, we can talk to etcd v3 server with the etcd client version shipped with the current docker volume plugin.  

If you need exact version of the etcd client (python module) -- this is the information
python-etcd==0.4.5

https://github.com/hpe-storage/python-hpedockerplugin/blob/plugin_v2/requirements-py3.txt#L69
Yes, our source is open source too.

**Comment by falenn** [comment]
Excellent, my mistake.  How are you leveraging the master branch?  Is plugin_v2 your master/release branch?  Sorry for the confusion
